src/platforms/node/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -15,7 +15,9 @@ Read more about [Custom Instrumentation](./../custom-instrumentation/) to learn
 
 <Note>
 
-If you’re adopting Performance in a high-throughput environment, we recommend testing prior to deployment to ensure that your service’s performance characteristics maintain expectations.
+If you're adopting Performance in a high-throughput environment, we recommend testing prior to deployment to ensure that your service's performance characteristics maintain expectations.
+
+Note that calling `Sentry.autoDiscoverNodePerformanceMonitoringIntegrations` might degrade startup performance of your app. This is especially a concern in Serverless functions. If you're experiencing this, manually add [database integrations](../../database/) of your choice instead (see example below).
```

Comment:
Also, isn't it pretty much 100% guaranteed that this function is slow?

Reply:
Yes I think so. Let me reword this slightly.